### DISCUSSION TITLE
tiles do not allow scroll page

### DISCUSSION THREAD

**Discussion by AZ77-Git** [olton/Metro-UI-CSS#1803] [Q&A]
Hello,

I have used the code bellow and i am encountring an issue.

the page refuse to allow scrolling when the page size change to a low size.

You can find the code in this codepen page https://codepen.io/AZ77/pen/MWmQEpP.

Have you an idea to find out how to let us scroll to display the end of the page ? 

Thank you very much.

**Comment by olton** [comment]
You are using position fixed, read about CSS property `position` and position value `fixed` to solve your problem.

**Reply by AZ77-Git** [reply]
Hello, thank you, i can imagine how it can work.

I have removed the fixed position first. but the div went on top of another object at the bottom.

Then i removed the pos-center, now the div is having a classical behaviour.

Can you help me to find out how to make a center position with this div ? 

I have saved the change on the codepen

Thank you.

**Comment by olton** [comment]
You can use a flex-box model for parent container:
```
<div class="parent">
    <div id="centered-element">...</div>
</div>
```

```
.parent {
    display: flex;
    justify-content: center;
    align-items: center;
}
```

**Reply by AZ77-Git** [reply]
It's working properly.

Thank you very much.

You can find the result on the codepen.

How can I close the subject and the issue ?
